one_day_intern/exceptions.py

```diff
@@ -6,6 +6,10 @@ class InvalidAssignmentRegistration(Exception):
     pass
 
 
+class InvalidInteractiveQuizRegistration(Exception):
```

Comment:
Hello, Indi. To ensure that the exception will be associated with a 400 response, can you please inherit the InvalidRequestException class instead of Exception? Thank you.

Reply:
Okay I'll refactor that jo

Reply:
<img width="555" alt="Screen Shot 2022-10-26 at 15 12 39" src="https://user-images.githubusercontent.com/70441226/197971476-46704f1d-8c69-4f68-a389-929de5bbd44b.png">
Hi, Jo, is this what you meant?

Reply:
Yes, Indi. The reason is the fact that we only put InvalidRequestException in the exception_config.py. In addition, it is still part of an invalid request and hence should yield response 400.